net-ftp/yafc/yafc-1.3.7-r2.ebuild

```diff
@@ -0,0 +1,51 @@
+# Copyright 1999-2022 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+inherit autotools bash-completion-r1
+
+DESCRIPTION="Console ftp client with a lot of nifty features"
+HOMEPAGE="https://github.com/sebastinas/yafc"
+SRC_URI="https://github.com/sebastinas/yafc/archive/v${PV}.tar.gz -> ${P}.tar.gz"
+
+LICENSE="GPL-2"
+SLOT="0"
+KEYWORDS="~amd64 ~arm ~ppc ~sparc ~x86"
+IUSE="ipv6 kerberos readline socks5 ssh"
+
+RDEPEND="
+	sys-libs/ncurses:=
+	dev-libs/libbsd
+	dev-libs/openssl:0=
+	kerberos? ( virtual/krb5 )
+	readline? ( >=sys-libs/readline-6:0= )
+	socks5? ( net-proxy/dante )
+	ssh? ( net-libs/libssh )
+"
+DEPEND="${RDEPEND}"
+
+DOCS=( BUGS NEWS README.md THANKS TODO )
+
+PATCHES=( "${FILESDIR}/${P}-fix-autoreconf.patch" )
+
+src_prepare() {
+	default
+	eautoreconf
+}
+
+src_configure() {
+	export ac_cv_ipv6=$(usex ipv6)
```

Comment:
IIRC we're trying to force ipv6 on whenever it doesn't require e.g. extra deps or other weirdness.